src/assessmentForm.ts

```diff
@@ -1,5 +1,7 @@
 /** AssessmentForm status types */
 export enum AssessmentFormStatus {
+  /** The form has been created but has not been shared with assignees */
+  Draft = 'DRAFT',
```

Comment:
Don't we need a migration for this?

Reply:
We will, but only after it merges to privacy-types I think. Then I'll update the package version in the monorepo and write a migration.

Reply:
gotcha! Looked too quickly and didn't notice this was privacy-types!